.github/scripts/.helm-tests/default/result.yaml

```
@@ -158,135 +158,162 @@ spec:
-          metadata:
-            type: object
-          spec:
-            description: AnalysisSpec defines the desired state of Analysis
-            properties:
-              analysisDefinition:
-                description: AnalysisDefinition refers to the AnalysisDefinition,
-                  a CRD that stores the AnalysisValuesTemplates
-                properties:
-                  name:
-                    description: Name defines the name of the referenced object
-                    type: string
-                  namespace:
-                    description: Namespace defines the namespace of the referenced
-                      object
+              type: string
+            metadata:
+              type: object
+            spec:
+              description: AnalysisSpec defines the desired state of Analysis
+              properties:
+                analysisDefinition:
+                  description: AnalysisDefinition refers to the AnalysisDefinition,
+                    a CRD that stores the AnalysisValuesTemplates
+                  properties:
+                    name:
+                      description: Name defines the name of the referenced object
+                      type: string
+                    namespace:
+                      description: Namespace defines the namespace of the referenced
+                        object
+                      type: string
+                  required:
+                    - name
+                  type: object
+                args:
+                  additionalProperties:
                     type: string
-                required:
-                - name
-                type: object
-              args:
-                additionalProperties:
+                  description: Args corresponds to a map of key/value pairs that can
+                    be used to substitute placeholders in the AnalysisValueTemplate
+                    query. i.e. for args foo:bar the query could be "query:percentile(95)?scope=tag(my_foo_label:)".
+                  type: object
+                timeframe:
+                  description: Timeframe specifies the range for the corresponding query
+                    in the AnalysisValueTemplate. Please note that either a combination
+                    of 'from' and 'to' or the 'recent' property may be set. If this
+                    is not the case, the Analysis can not be added to the cluster.
+                  properties:
+                    from:
+                      description: From is the time of start for the query, this field
+                        follows RFC3339 time format
+                      format: date-time
+                      type: string
+                    recent:
+                      description: Recent describes a recent timeframe using a duration
+                        string. E.g. Setting this to '5m' will cause the Analysis to
+                        be performed for the last five minutes
```

```suggestion
                        string. E.g. Setting this to '5m' causes the Analysis to
                        be performed for the last five minutes
```
Could it say, "Setting this to `5m` provides an Analysis for the last five minutes" ?